From c5b9c082dafe2ad2ac118ffd39c5b88307f3357f Mon Sep 17 00:00:00 2001
From: Ashok Sidipotu <ashok.sidipotu@collabora.com>
Date: Thu, 24 Feb 2022 16:12:23 +0530
Subject: [PATCH 08/12] Revert "meson: declare spa_dep and
 override_dependency() for spa and pipewire"

This reverts commit 7ab5c35cadc075bcdf44031f1f7d95352cf9a13b.
---
 src/pipewire/meson.build | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/src/pipewire/meson.build b/src/pipewire/meson.build
index 5cfaba3f9..2dd5d790e 100644
--- a/src/pipewire/meson.build
+++ b/src/pipewire/meson.build
@@ -121,11 +121,6 @@ pkgconfig.generate(libpipewire,
   description : 'PipeWire Interface',
   version : pipewire_version,
   extra_cflags : '-D_REENTRANT',
-  variables : ['moduledir=${libdir}/@0@'.format(pipewire_name)],
-  uninstalled_variables : [
-    'moduledir=${prefix}/src/modules',
-    'confdatadir=${prefix}/src/daemon',
-  ],
-)
+  variables : ['moduledir=${libdir}/@0@'.format(pipewire_name)])
 
 subdir('extensions')
-- 
2.35.1

